electron/build/win32/strings/installer.en.isl

```diff
@@ -0,0 +1,3 @@
+[CustomMessages]
```

Comment:
Installer strings that will need to be localized. Localization of this will be annoying, we'll need to figure it out.